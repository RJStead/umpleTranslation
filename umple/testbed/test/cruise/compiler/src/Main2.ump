generate Java "../src-gen-umple";

class Main2
{
    public static void main(String [] argv){
        int a = "not an int";
        int b = 10;
        b++;
        a += b;
    }
}
