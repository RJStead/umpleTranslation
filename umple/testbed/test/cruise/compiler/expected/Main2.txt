Main2.ump:6: error: incompatible types: String cannot be converted to int
    int a = "not an int";
            ^
1 error
